kuma/static/js/analytics.js

```diff
@@ -93,20 +93,21 @@
         trackClientErrors: function() {
             $(win).on('error', function(e) {
                 var originalEvent = e.originalEvent;
-                analytics.trackError(' JavaScript Error: ' + originalEvent.message + ' ; ' + originalEvent.filename + ':' + originalEvent.lineno);
+                analytics.trackError('JavaScript Error', win.location.href, originalEvent.message + ' ; ' + originalEvent.filename + ':' + originalEvent.lineno);
             });
             $(doc).ajaxError(function(e, request, settings) {
-                analytics.trackError('AJAX Error: ' +  settings.url + ' : ' + e.result);
+                analytics.trackError('AJAX Error', win.location.href, settings.url + ' : ' + e.result);
```

Comment:
Same comment here - we could put the `settings.url` value into the `action` argument if we want to report by that dimension.